dotcom-rendering/makefile

```diff
@@ -32,7 +32,7 @@ deploy:
 
 build: clean-dist install
 	$(call log, "building production bundles")
-	@NODE_ENV=production webpack --config ./scripts/webpack/webpack.config.js
+	@NODE_ENV=production webpack --config ./scripts/webpack/webpack.config.js --progress
```

Comment:
Just noting here that prior to this change, no feedback on the progress (or any errors) of webpack were given during the production build process.